.pre-commit-config.yaml

```diff
@@ -42,6 +42,15 @@ repos:
         language: system
         types: [python]
 
+  - repo: local
```

Comment:
We don't actually want this to run as local, but instead against the repo. Try replacing the pre-commit in Check Run or another target repo with this one and refactor the addresses to work with the remote, more or less like this (which I'm sure is incorrect)

```yaml
  - repo: https://github.com/agritheory/test_utils
    rev: v2.2.6
    hooks:
      - id: validate_javascript_dependencies
        always_run: true
        name: pre-commit/validate_javascript_dependencies.py
        entry: python3 pre-commit/validate_javascript_dependencies.py
        language: system
        types: [python]
      - id: validate_python_dependencies
        always_run: true
        name: pre-commit/validate_python_dependencies.py
        entry: python3 pre-commit/validate_python_dependencies.py
        language: system
        types: [python]
      - id: validate_customizations
        always_run: true
        name: pre-commit/validate_customizations.py
        entry: python3 pre-commit/validate_customizations.py
        language: system
        types: [python]
```

Reply:
The rev in that note is also wrong

Reply:
> We don't actually want this to run as local, but instead against the repo. Try replacing the pre-commit in Check Run or another target repo with this one and refactor the addresses to work with the remote, more or less like this (which I'm sure is incorrect)
> 
> ```yaml
>   - repo: https://github.com/agritheory/test_utils
>     rev: v2.2.6
>     hooks:
>       - id: validate_javascript_dependencies
>         always_run: true
>         name: pre-commit/validate_javascript_dependencies.py
>         entry: python3 pre-commit/validate_javascript_dependencies.py
>         language: system
>         types: [python]
>       - id: validate_python_dependencies
>         always_run: true
>         name: pre-commit/validate_python_dependencies.py
>         entry: python3 pre-commit/validate_python_dependencies.py
>         language: system
>         types: [python]
>       - id: validate_customizations
>         always_run: true
>         name: pre-commit/validate_customizations.py
>         entry: python3 pre-commit/validate_customizations.py
>         language: system
>         types: [python]
> ```

Yes, I had a plan once we complete all the pre-commit configurations then we can start putting this as a pre-commit hook in other repos. I'll make the change.

Reply:
@MyuddinKhatri OK that makes sense. Let's do that after the rest of the pre-commit scripts are done.